src/app/plugins/garage.plugin.ts

```diff
@@ -0,0 +1,86 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, IHttpResponse } from '../../common/types';
+
+const cheerio = require("cheerio")
+
+class Garage {
+	public name : string = "";
+	public available: number = 0;
+	public saturation: number = 0;
+	public capacity: number = 0;
+	public percent_full: number = 0;
+	public percent_avail: number = 0;
+}
+
+export class GaragePlugin extends Plugin {
+	public name: string = 'Garage Plugin';
+	public description: string = 'Get garage status';
+	public usage: string = 'garage [which garage]';
+	public permission: ChannelType = ChannelType.Public;
+
+	private _API_URL: string = "http://secure.parking.ucf.edu/GarageCount/iframe.aspx";
+	private title_message: string = "**Current UCF Garage Saturation**"
+
+	public process_response(text: String): Garage[] {
+		const $ = cheerio.load(text)
+		const garages = $(".dxgv")
+
+		var last = ""
```

Comment:
We typically never use `var` when declaring a variable. A good rule of thumb is to always use `const` unless you are actually modifying the value of the variable. In that situation, you'll want to be using `let` instead. I recommend you to download ESLint and Prettier and setup the linters to apply the lints on-save of file. It will make everything flow smoother.